R/type_histogram.R

```diff
@@ -39,7 +39,7 @@ type_hist = type_histogram
 data_histogram = function(breaks = "Sturges") {
     hbreaks = breaks
     fun = function(by, facet, ylab, col, bg, ribbon.alpha, datapoints, .breaks = hbreaks, ...) {
-        hbreaks = ifelse(!is.null(.breaks), .breaks, "Sturges")
+        hbreaks = ifelse(!mapply(is.null, .breaks), .breaks, "Sturges")
```

Comment:
`mapply` is nice and concise. But it's more efficient (safer) to use something like:

```r
!vapply(.breaks, function(x) is.null(x), FUN.VALUE = logical(1)
```
Or, perhaps even:

```r
any(lengths(.breaks) == 0)
```

Reply:
Come to think of it, couldn't we just check the first element of `.breaks?` I'm trying to think of situations where the first element would be `NULL` but the others not (or, vice verse)....